### PR TITLE
fix(linter): address gosimple linter reports & errors

### PR DESCRIPTION
**Pull request by skrdgraph**
GoLinter failures: 
- related to https://github.com/dgraph-io/dgraph/pull/8620 & https://github.com/dgraph-io/dgraph/actions/runs/3971511349/jobs/6815792372
- Addressing this partially https://github.com/dgraph-io/dgraph/pull/8572#issuecomment-1401541634 
- fix `gosimple` & `unused` linter reports
- comment the rest to address later

### CODE REVIEW

**Comment by skrdgraph** [comment]
@anurags92 @mangalaman93 @harshil-goel, I have uncommented and fixed 2 linters. We will gradually open up the rest.

**Review by mangalaman93 — APPROVED** [review]
This is a better idea. LGTM

File: .golangci.yml [hunk 1]
```diff
@@ -19,11 +19,11 @@ linters-settings:
 linters:
   disable-all: true
   enable:
-    - errcheck
-    - ineffassign
-    - gas
+    #- errcheck
```
Comment (anurags92) [inline .golangci.yml:22]: Is this disabling them or enabling them? I am confused.

Comment (skrdgraph) [reply .golangci.yml:22]: as part of this PR https://github.com/dgraph-io/dgraph/pull/8620/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9L22-R31 .. we tried addressing all the linters by uncommenting it. 

The rebase issue hit us in that PR. The linter ran fine on that particular PR, but failed after a merge to `main`. This PR is partially fixing errors from 2 linters & taking things back to what it was before.

Comment (anurags92) [reply .golangci.yml:22]: Understood.

**Pull request merged**